cvem/connectors/one/OpenNebula.py

```diff
@@ -148,10 +148,10 @@ def get_vm_list():
 			res = []
 			for vm in res_vm.VM:
 				host = HostInfo(int(vm.HISTORY_RECORDS.HISTORY[0].HID), vm.HISTORY_RECORDS.HISTORY[0].HOSTNAME)
-				new_vm = VirtualMachineInfo(int(vm.ID), host, vm.TEMPLATE.MEMORY * 1024, vm)
+				new_vm = VirtualMachineInfo(int(vm.ID), host, int(vm.MAXMEMORY), vm)
```

Comment:
The VM.allocated_memory is the amount of memory originally allocated to this VM. So this value is the MEMORY field in the original TEMPLATE. Why do you think that is better to use MAXMEMORY?